### ISSUE TITLE
error while trying to convert some specific websites

### ISSUE THREAD

**Issue by dfernan** [wkhtmltopdf/wkhtmltopdf#3950]
E.g., 
```
wkhtmltopdf --javascript-delay 1000000 --debug-javascript 'https://www.lowes.com/pl/Electrical-wire-cable-Electrical/4294722493' lowes.pdf
Error: Failed loading page https://www.lowes.com/pl/Electrical-wire-cable-Electrical/4294722493 (sometimes it will work just to ignore this error with --load-error-handling ignore)
Exit with code 1, due to unknown error.
```
I am using version wkhtmltopdf 0.12.2.4

It actually fails for all pages in lowes.com.

If I do ignore it renders a pdf with almost no content.  I tried w/o javascript options and it also doesn't work.

**Comment by alloylab** [comment]
what os? Is it a patched qt version? have you tried 0.12.5?

**Comment by alloylab** [comment]
We are closing this out since we haven't heard from you. If you are still having this issue please feel free to re-open it. Thanks.
